### PR TITLE
Fix up changelog, readme, audit

### LINKED ISSUE #152: The cache example in the README should use actions/cache@v2
`v2` has been out since May 2020 and works with the caching example in the README

### PR DESCRIPTION
**Pull request by bollwyvl**
Docs
- [x] fixes #152 
- [x] updates changelog

Deps
- [x] resolve `package-lock.json`
- [x] apply `npm audit` (one `minor` remains, investigating...)
- [x] randomly pick one of #86 or #82 unless somebody weighs in...

### CODE REVIEW

**Comment by bollwyvl** [comment]
:bell: @conda-incubator/setup-miniconda-team

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,23 +1,53 @@
...
+[#126]: https://github.com/conda-incubator/setup-miniconda/pull/126
+[#127]: https://github.com/conda-incubator/setup-miniconda/pull/127
+[#130]: https://github.com/conda-incubator/setup-miniconda/pull/130
+[#133]: https://github.com/conda-incubator/setup-miniconda/pull/133
+[#137]: https://github.com/conda-incubator/setup-miniconda/pull/137
+[#138]: https://github.com/conda-incubator/setup-miniconda/pull/138
+[#140]: https://github.com/conda-incubator/setup-miniconda/pull/140
+[#148]: https://github.com/conda-incubator/setup-miniconda/pull/148
+
+## [Miniforge]: https://github.com/conda-forge/miniforge
```
Comment (jaimergp) [inline CHANGELOG.md:40]: I am not sure this line belongs here?

Comment (bollwyvl) [reply CHANGELOG.md]: thanks, good catch!

**Review by goanpeca — APPROVED** [review]

**Comment by bollwyvl** [comment]
Thanks!

**Pull request merged**